src/App.js

```diff
@@ -1,7 +1,77 @@
+import React from 'react';
 import "./App.css";
+import ProductData from './data/productData';
+import Products from './Components/Products';
+import AddToCart from './Components/AddToCart';
+import Checkout from './Components/Checkout';
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      cart: [],
+      subtotal: '',
+      tax: '',
+      total: ''
+    };
+  };
+
+  addProduct = (id) => {
+    ProductData.map((product, i) => {
+      if(id === product.id) {
+        this.state.cart.push({product})
+      }
+    });
+    this.setState({
+      cart: this.state.cart
+    })
+  };
+
+  subTotal = () => {
```

Comment:
subTotal, productTax, and total can be separate helper functions instead of methods of the App class because they shouldn't need to update the state of App.